recipes/poco/all/patches/1.9.1-2.patch

```diff
@@ -0,0 +1,273 @@
+--- ApacheConnector/CMakeLists.txt
++++ ApacheConnector/CMakeLists.txt
+@@ -10,8 +10,8 @@
+ 
+ # Version Resource
+ if(MSVC AND NOT POCO_STATIC)
+-    source_group("Resources" FILES ${PROJECT_SOURCE_DIR}/DLLVersion.rc)
+-    list(APPEND SRCS ${PROJECT_SOURCE_DIR}/DLLVersion.rc)
++    source_group("Resources" FILES ${CMAKE_SOURCE_DIR}/DLLVersion.rc)
++    list(APPEND SRCS ${CMAKE_SOURCE_DIR}/DLLVersion.rc)
```

Comment:
`PROJECT_SOURCE_DIR` should contain the same value with and without the _Cmake wrapper_ project, as Poco is defining its own project afterward. why these changes?

Reply:
I don't remember what I was trying to accomplish/fix here...